provider/source/src/datetime/neo_skeleton.rs

```diff
@@ -213,6 +213,9 @@ impl SourceDataProvider {
             })
             .map(|trio| {
                 enforce_consistent_field_lengths(trio, |previous_field, field| {
+                    if attributes.as_str() == "ej" {
+                        return; // too noisy
```

Comment:
question: what's going on here?

Reply:
I changed it to filter by quality instead of a blanket filter on `"ej"`.